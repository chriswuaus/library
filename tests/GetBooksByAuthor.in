author yeehaw asdf
add collection test.csv
author yeet tetyet ytey
AUTHOR Homer
AUthor Bob Kane
author J.K. Rowling
add collection duplicates.csv
AUTHOR Homer
AUthor Bob Kane
author J.K. Rowling
exit
