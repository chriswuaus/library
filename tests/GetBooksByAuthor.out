user: No books in system.

user: 17 books successfully added.

user: No books by yeet tetyet ytey.

user: The Iliad (Homer)
The Odyssey (Homer)

user: Batman Volume 1 (Bob Kane)

user: Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: 14 books successfully added.

user: The Iliad (Homer)
The Odyssey (Homer)
The Iliad (Homer)
The Odyssey (Homer)

user: Batman Volume 1 (Bob Kane)
Batman Volume 1 (Bob Kane)

user: Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: Ending Library process.
